libraries/AP_SmartAudio/AP_SmartAudio.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include <AP_Common/AP_Common.h>
+#include <AP_Param/AP_Param.h>
+#include <AP_SerialManager/AP_SerialManager.h>
+#include "AP_SmartAudioDevice.h"
+
+class AP_SmartAudio
+{
+
+public:
+    AP_SmartAudio();
+
+    /* Do not allow copies */
+    AP_SmartAudio(const AP_SmartAudio &other) = delete;
+    AP_SmartAudio &operator=(const AP_SmartAudio&) = delete;
+
+    static const struct AP_Param::GroupInfo var_info[];
+
+    enum SmartAudioProtocol {
+        SmartAudioProtocol_NONE         = 0,
+        SmartAudioProtocol_V1_OneWay    = 1,
+        SmartAudioProtocol_V1           = 2,        
+        SmartAudioProtocol_V2_OneWay    = 3,
+        SmartAudioProtocol_V2           = 4,
+        SmartAudioProtocol_V21_OneWay   = 5,
+        SmartAudioProtocol_V21          = 6
+    };
+
+    enum SmartAudio_PowerMode {        
+        SmartAudio_Power_0,
```

Comment:
In the future, do the values of the value and the definition name become inconsistent? If there is, I think that it is better not to write a value in the definition name.

Reply:
Those are the names of power levels. Power_0 mean the minimal level, Power_4 and _5 mean maximal power level. The actual values are in the PWR_0 - PWR_5 parameters, they could be different between protocols or VTX model.